modules/xmpp/JingleSessionPC.js

```diff
@@ -1929,12 +1936,7 @@ export default class JingleSessionPC extends JingleSession {
 
                                 // Apply the cached video constraints on the new video sender.
                                 .then(() => this.peerconnection.setSenderVideoConstraint())
-                                .then(() => {
-                                    // Configure max bitrate on the video sender when media is routed through JVB.
-                                    if (!this.isP2P) {
-                                        return this.peerconnection.setMaxBitRate(newTrack);
-                                    }
-                                });
+                                .then(() => this.peerconnection.setMaxBitRate(newTrack));
```

Comment:
There should be no need for passing the `newTrack` argument, as the TPC instance should already have it in it's state.

This weirdness comes from the fact that currently TPC does not take care of it's responsibilities which is making sure that whatever value it has as max bitrate should be synchronized whenever there's change to the track senders it owns. Now this entire sync up chain should not be here, but I'll try to refactor it another day, so don't worry about it.